package.json

```diff
@@ -48,8 +47,8 @@
       "TestDelayOrError": "components/TestDelayOrError.coffee"
     },
     "graphs": {
-      "RepeatTest": "spec/fixtures/repeat.fbp",
-      "IIPKickTest": "spec/fixtures/kickiip.fbp"
+      "RepeatTest": "graphs/repeat.fbp",
```

Comment:
Why can't one have graphs whereever one would like with fbp-manifest?
This breaking change is also not mentioned in https://github.com/noflo/noflo/blob/master/CHANGES.md